### PR TITLE
[compute/cker] Remove UNUSED_RELEASE macro

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit removes UNUSED_RELEASE() macro in cker.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Refer #14336

### CODE REVIEW

File: compute/cker/include/cker/operation/reference/integer_ops/DepthwiseConvHybrid.h [hunk 1]
```diff
@@ -53,17 +53,15 @@ inline void DepthwiseConvHybridPerChannel(const DepthwiseConvParams &params,
...
   const int input_width = input_shape.Dims(2);
   const int input_depth = input_shape.Dims(3);
   const int filter_height = filter_shape.Dims(1);
   const int filter_width = filter_shape.Dims(2);
   const int output_height = output_shape.Dims(1);
   const int output_width = output_shape.Dims(2);
-  const int bias_depth = bias_shape.FlatSize();
-  UNUSED_RELEASE(output_depth);
-  UNUSED_RELEASE(bias_shape);
+  [[maybe_unused]] const int bias_depth = bias_shape.FlatSize();
```
Comment (seockho-kim) [inline compute/cker/include/cker/operation/reference/integer_ops/DepthwiseConvHybrid.h:64]: 'bias_shape' is in the arguments.

**Review by batcheu — APPROVED** [review]
LGTM

File: compute/cker/include/cker/operation/reference/integer_ops/DepthwiseConvHybrid.h (hunk 1 above)
Comment (hseok-oh) [reply compute/cker/include/cker/operation/reference/integer_ops/DepthwiseConvHybrid.h]: I've removed this change.

**Review by seockho-kim — APPROVED** [review]
LGTM

**Pull request merged**